inc/files.php

```diff
@@ -477,7 +480,7 @@ function ppom_uploaded_file_preview( $file_name, $settings ) {
 		// Tools group
 		$file_tools .= '<div class="btn-group" role="group" aria-label="Tools" style="text-align: center; display: block;">';
 		// $file_tools .= '<a href="#" class="nm-file-tools btn btn-primary u_i_c_tools_del" title="'.__('Remove', "woocommerce-product-addon").'"><span class="fa fa-times"></span></a>';
-		$file_tools .= '<a href="#" class="nm-file-tools btn btn-primary u_i_c_tools_del" title="' . __( 'Remove', 'woocommerce-product-addon' ) . '">' . __( 'Delete', 'woocommerce-product-addon' ) . '</span></a>';
+		$file_tools .= '<button href="#" class="nm-file-tools btn btn-primary u_i_c_tools_del" title="' . __( 'Remove', 'woocommerce-product-addon' ) . '">' . __( 'Delete', 'woocommerce-product-addon' ) . '</span></button>';
```

Comment:
You missed the href tag in the button. 😄

Reply:
Also I see there's a closing `</span>` without an opening. I think in past it was for icons but at some point we removed it but forgot to remove the closing tag.